packages/core/src/card/card.element.spec.ts

```diff
@@ -0,0 +1,26 @@
+import '@cds/core/card/register.js';
+import { html } from 'lit-html';
+import { componentIsStable, createTestElement, removeTestElement } from '@cds/core/test';
+import { CdsCard } from '@cds/core/card';
+
+describe('Card element: ', () => {
+  let testElement: HTMLElement;
+  let component: CdsCard;
+  const placeholderText = 'Placeholder';
+
+  afterEach(() => {
+    removeTestElement(testElement);
+  });
+
+  describe('create the card component', () => {
+    beforeEach(async () => {
+      testElement = await createTestElement(html`<cds-card>${placeholderText}</cds-card>`);
+      component = testElement.querySelector<CdsCard>('cds-card');
+    });
+
+    it('should create the component', async () => {
+      await componentIsStable(component);
+      expect(component.innerText).toBe(placeholderText);
```

Comment:
This can be simplified to `expect(component).toBeTruthy();`

If you also want to test the projection a second `it('projects the content', () => {})` could be added.